src/Scaffolding/VS.Web.CG.Mvc/Templates/Identity/Bootstrap4/wwwroot/css/site.css

```diff
@@ -1,23 +1,22 @@
 ﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification\
 for details on configuring this project to bundle and minify static web assets. */
 
-a.navbar-brand {
-  white-space: normal;
-  text-align: center;
-  word-break: break-all;
-}
-
-/* Sticky footer styles
--------------------------------------------------- */
 html {
   font-size: 14px;
 }
-@media (min-width: 768px) {
+
+@media (min-width: 576px) {
```

Comment:
because the layout uses bootstrap small size which is 576
if you want to use 768 here, the layout has to be changed to use large bootstrap breakpoints